src/main/java/pvs/app/controller/ProjectController.java

```diff
@@ -57,12 +55,21 @@ public ProjectController(ProjectService projectService, RepositoryService reposi
         }
     }
 
+    @GetMapping("/repository/gitlab/check")
+    public ResponseEntity<String> checkGitlabURL(@RequestParam("url") String url) {
+        if (repositoryService.checkGitlabURL(url)) {
+            return ResponseEntity.status(HttpStatus.OK).body(successMessage);
+        } else {
+            return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(urlInvalidMessage);
+        }
```

Comment:
Please unwrap the `else { ... }` since you have already returned in above.
```suggestion
        }
        return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(urlInvalidMessage);
```